README.md

```diff
@@ -1,13 +1,6 @@
-# Medplum Hello World
+# Foo Clinical
 
-This example app demonstrates the following:
-
-- Creating a new React app with Vite and TypeScript
-- Adding Medplum dependencies
-- Adding basic URL routing
-- Using the Medplum client to search for FHIR resources
-- Using Medplum GraphQL queries
-- Using Medplum React controls to display FHIR data
+This is a sample EHR-like application using Medplum. Medplum is a developer platform for medical apps.
 
 ## Getting Started
```

Comment:
I really like that Medplum included a folder structure. Not enough projects do this. https://github.com/medplum/medplum#folder-structure

Reply:
For FooClinical, I currently only have a `src` folder which only has a `pages` folder inside it. Everything else is just single files, other than my `dist` and `node_modules` folder, which aren't included in Medplum's structure. Since there are so few folders, I'm not sure if it makes sense to include this yet. If it starts getting more complicated as I add more, then I agree it could be added then.